sycl/source/detail/context_impl.cpp

```diff
@@ -242,6 +242,40 @@ context_impl::get_info<info::context::atomic_fence_scope_capabilities>() const {
   return CapabilityList;
 }
 
+template <>
+std::string context_impl::get_backend_info<info::platform::version>() const {
```

Comment:
Should we also use some traits to encode return type instead of hardcoding it here? This way if it is ever changed, it would be enough to change it just once in a single place